ci build stage failure

https://circleci.com/gh/vapor-ware/synse-cli/384

```
#!/bin/bash -eo pipefail
cd cmd/
gox --output="../build/${BINARY_NAME}_{{.OS}}_{{.Arch}}" -osarch='!darwin/386'

Number of parallel builds: 31

-->    linux/mipsle: github.com/vapor-ware/synse-cli/cmd
-->       linux/arm: github.com/vapor-ware/synse-cli/cmd
-->     windows/386: github.com/vapor-ware/synse-cli/cmd
-->      netbsd/arm: github.com/vapor-ware/synse-cli/cmd
-->       linux/386: github.com/vapor-ware/synse-cli/cmd
-->     linux/amd64: github.com/vapor-ware/synse-cli/cmd
-->     freebsd/386: github.com/vapor-ware/synse-cli/cmd
-->     openbsd/386: github.com/vapor-ware/synse-cli/cmd
-->   freebsd/amd64: github.com/vapor-ware/synse-cli/cmd
-->   windows/amd64: github.com/vapor-ware/synse-cli/cmd
-->   openbsd/amd64: github.com/vapor-ware/synse-cli/cmd
-->     linux/s390x: github.com/vapor-ware/synse-cli/cmd
-->    darwin/amd64: github.com/vapor-ware/synse-cli/cmd
-->     freebsd/arm: github.com/vapor-ware/synse-cli/cmd
-->    netbsd/amd64: github.com/vapor-ware/synse-cli/cmd
-->  linux/mips64le: github.com/vapor-ware/synse-cli/cmd
-->      linux/mips: github.com/vapor-ware/synse-cli/cmd
-->      netbsd/386: github.com/vapor-ware/synse-cli/cmd
-->    linux/mips64: github.com/vapor-ware/synse-cli/cmd

6 errors occurred:
--> linux/386 error: exit status 2
Stderr: # vendor/golang_org/x/crypto/cryptobyte
compile: writing output: write $WORK/b088/_pkg_.a: cannot allocate memory
go build net: /usr/local/go/pkg/tool/linux_amd64/compile: signal: killed

--> freebsd/arm error: exit status 2
Stderr: # encoding/hex
compile: writing output: write $WORK/b085/_pkg_.a: cannot allocate memory

--> linux/mips64 error: exit status 2
Stderr: # github.com/vapor-ware/synse-cli/vendor/github.com/hashicorp/hcl/hcl/parser
compile: writing output: write $WORK/b153/_pkg_.a: cannot allocate memory
# vendor/golang_org/x/text/unicode/norm
compile: writing output: write $WORK/b102/_pkg_.a: cannot allocate memory

--> freebsd/amd64 error: exit status 2
Stderr: # vendor/golang_org/x/text/unicode/norm
/usr/local/go/src/vendor/golang_org/x/text/unicode/norm/tables.go:5146:17: malformed integer constant: 0x
/usr/local/go/src/vendor/golang_org/x/text/unicode/norm/tables.go:5146:19: read /usr/local/go/src/vendor/golang_org/x/text/unicode/norm/tables.go: cannot allocate memory
/usr/local/go/src/vendor/golang_org/x/text/unicode/norm/tables.go:5146:19: malformed hex constant
/usr/local/go/src/vendor/golang_org/x/text/unicode/norm/tables.go:5146:19: read /usr/local/go/src/vendor/golang_org/x/text/unicode/norm/tables.go: cannot allocate memory
/usr/local/go/src/vendor/golang_org/x/text/unicode/norm/tables.go:5146:19: syntax error: unexpected EOF, expecting comma or }

--> darwin/amd64 error: exit status 2
Stderr: # vendor/golang_org/x/text/unicode/bidi
compile: writing output: write $WORK/b104/_pkg_.a: cannot allocate memory

--> openbsd/amd64 error: exit status 1
Stderr: go build github.com/vapor-ware/synse-cli/vendor/github.com/aws/aws-sdk-go/aws/endpoints: /usr/local/go/pkg/tool/linux_amd64/compile: signal: killed

Exited with code 1
```


not sure what changed to cause this. likely a dependency thing, since the ci build where this is failing is for a spelling correction..

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -81,7 +81,7 @@ jobs:
           name: "Building"
           command: |
             cd cmd/
-            gox --output="../build/${BINARY_NAME}_{{.OS}}_{{.Arch}}" -osarch='!darwin/386'
+            gox --output="../build/${BINARY_NAME}_{{.OS}}_{{.Arch}}" -osarch='!darwin/386' -parallel=10
 
       - save_cache:
           when: on_success

